content/en/blog/_posts/2024-12-11-Kubernetes-v1-32-Release.md

```diff
@@ -0,0 +1,301 @@
+
+The Kubernetes scheduler has been enhanced with Asynchronous Preemption, a feature that improves scheduling throughput by handling preemption operations asynchronously. Preemption ensures higher-priority pods get the resources they need by evicting lower-priority ones, but this process previously involved blocking operations like PostFilter execution, slowing down the scheduler. With this enhancement, such tasks are now processed in parallel, allowing the scheduler to continue scheduling other pods without delays.
+
+This work was done as a part of KEP [#4832](https://github.com/kubernetes/enhancements/issues/4832) by [SIG Scheduling](https://github.com/kubernetes/community/tree/master/sig-scheduling).
+
+ 
+### Mutating Admission Policies Using CEL expressions
+
+This feature leverages CEL's object instantiation and JSON Patch strategies, combined with Server Side Apply’s merge algorithms. It simplifies policy definition, reduces mutation conflicts, and enhances admission control performance while laying a foundation for more robust, extensible policy frameworks in Kubernetes.
+
+The Kubernetes scheduler now supports CEL-based Mutating Admission Policies, providing a lightweight, efficient alternative to mutating admission webhooks. With this enhancement, administrators can use the Common Expression Language (CEL) to declare mutations like setting labels, defaulting fields, or injecting sidecars with simple, declarative expressions. This approach reduces operational complexity, eliminates the need for webhooks, and integrates directly with the kube-apiserver, offering faster and more reliable in-process mutation handling.
+
+This work was done as a part of [KEP #3962](https://github.com/kubernetes/enhancements/issues/3962) by [SIG API Machinery](https://github.com/kubernetes/community/tree/master/sig-api-machinery).
+
+
+### Pod-Level Resource Specifications 
+    
+This enhancement introduces a simplified way to manage resources in Kubernetes by allowing users to define resource requests and limits for an entire pod instead of specifying them individually for each container. This creates a shared resource pool that all containers in the pod can dynamically utilize, optimizing resource usage and reducing over-provisioning, especially for workloads with varying or bursty resource demands.
+
+With this feature, users gain more flexibility and efficiency in resource allocation, as the pod-level settings complement existing container-level specifications. By directly leveraging pod-level cgroup settings, Kubernetes ensures better performance for tightly coupled applications while maintaining backward compatibility with container-level configurations. This enhancement represents a significant step forward in simplifying resource management for multi-container pods.
+
+This work was done as part of [KEP #4678](https://github.com/kubernetes/enhancements/issues/4678) by [SIG Node](https://github.com/kubernetes/community/tree/master/sig-node).
```

Comment:
The KEP is https://github.com/kubernetes/enhancements/issues/2837, https://github.com/kubernetes/enhancements/issues/4678 is the PR it seems.

```suggestion
This work was done as part of [KEP #2837](https://github.com/kubernetes/enhancements/issues/2837) by [SIG Node](https://github.com/kubernetes/community/tree/master/sig-node).
```